invenio_global_search/proxies.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 Graz University of Technology.
+#
+# invenio-global-search is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Helper proxy to the state object."""
+
+from flask import current_app
+from werkzeug.local import LocalProxy
+
+current_records_dublin_core = LocalProxy(
+    lambda: current_app.extensions["invenio-global-search"],
```

Comment:
This is a bit confusing and mixing concerns of the modules.
This should be at the `dublin-core` module. A `current_global_search` fits better here.

Reply:
yes, that could be a copy paste error!